versions.json

```diff
@@ -55,7 +55,7 @@
     "lts": "2022-10-25",
     "maintenance": "2023-10-18",
     "end": "2025-04-30",
-    "codename": "",
```

Comment:
Per https://github.com/nodejs/docker-node/pull/1795#pullrequestreview-1155777601, maybe this line is the only change should be existed here?

Reply:
The automated PRs are still broken right now because of the key changes. They'll keep being "dirty" and needing manual PRs till we get another v14 release